### PR TITLE
Wille/dashboardfixes

### PR DESCRIPTION
**Pull request by willeasp**
- fixed small bugs
- added a refresh button
- added a loadingsymbol when you click refresh and deleting etc

### CODE REVIEW

**Comment by willeasp** [comment]
fixes #25

File: webapp/src/cosmos/cosmos.js [hunk 1]
```diff
@@ -91,9 +91,15 @@ const cosmos = {
...
+
+        const testQuery = {
+            query: "SELECT c.id, c.deviceId, c.name, c.location, c.maxDist, c.minDist FROM c"
+        }
         const { resources: devices } = await deviceContainer.items
-        .query(deviceQuery)
+        .query(testQuery)
         .fetchAll();   
+
+        console.log(devices);
```
Comment (Joppevalck) [inline webapp/src/cosmos/cosmos.js:102]: log

File: webapp/src/views/Dashboard.vue [hunk 2]
```diff
@@ -385,11 +396,20 @@
...
-          cosmos.deleteDevice(device)
-          .then(this.getAllDevices());
+          await cosmos.deleteDevice(device);
+          await new Promise(resolve => setTimeout(resolve, 800));
+          this.getAllDevices();
         }
+      },
+      refresh() {
+        // added to make loading symbol stay active longer
+        console.log("refresh")
```
Comment (Joppevalck) [inline webapp/src/views/Dashboard.vue:409]: log

**Review by Joppevalck — APPROVED** [review]

**Review by joakgus — APPROVED** [review]
looks good

**Pull request merged**